### ISSUE TITLE
asking for password

### ISSUE THREAD

**Issue by jerryivanhoe** [bigdatagenomics/eggo#130]
Hi, I ran the script @ AWS

[ec2-user@ip-172-31-13-26 ~]$ eggo-cluster provision --region us-east-1 --launch
Stack 'bdg-eggo' already exists. Reusing.
Creating launcher instance.
Waiting for instance to enter 'running' state.....................
Instance is now in 'running' state. Waited 334 seconds.
[54.175.240.90] Executing task 'install_director_client'
[54.175.240.90] sudo: wget http://archive.cloudera.com/director/redhat/6/x86_64/
[54.175.240.90] Login password for 'ec2-user':
Sorry, you can't enter an empty password. Please try again.
[54.175.240.90] Login password for 'ec2-user':
[54.175.240.90] Login password for 'ec2-user':
[54.175.240.90] Login password for 'ec2-user':
[54.175.240.90] Login password for 'ec2-user':
This user has no password 
I tried to set one - and same problem -it is not accepted

greetings
Jerry

**Comment by laserson** [comment]
Ok, this is stupid on my part, but I should get rid of the default for the stack-name.  You should also add a `--stack-name my-stack` option to the command.

**Comment by laserson** [comment]
Also, do you have an EC2 key pair set up?

**Comment by jerryivanhoe** [comment]
Hi again, 
I setup an EC2 instance and installed eggo. Then I set my ENV and copied the pem file to the local dir, setting mod to 600. (s pics). Finally I started this command:
eggo-cluster provision --region us-east-1 –stack-name eggo --launcher-instance-type m3.medium --availability-zone us-east-1a -n 4
The cloud Formation stack runs w/o error and the instance starts successfully. Then the password problem shows up. 
any idea what's wrong with my settings ?
thanks a lot in advance
-Jerry
![pic1](https://cloud.githubusercontent.com/assets/13957285/10862760/960a3468-7fb6-11e5-9571-8af9aeae2d6b.png)
![pic2](https://cloud.githubusercontent.com/assets/13957285/10862759/96075770-7fb6-11e5-9a37-745ded8e4eba.png)
![pic3](https://cloud.githubusercontent.com/assets/13957285/10862761/960d77cc-7fb6-11e5-813e-9763a09e15f6.png)
![pic4](https://cloud.githubusercontent.com/assets/13957285/10862758/96063b10-7fb6-11e5-8e32-f14a0f2b202e.png)
